README.md

```diff
@@ -15,6 +15,57 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
   * Gluesync  - A metastore event listener to replay Hive metadata events in a Glue catalog.
   * Metastore authorization - A metastore pre-event listener to handle authorization using Ranger.
 
+## Variables
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| apiary_customer_accounts | aws account ids for clients of this metastore | list | - | yes |
+| apiary_database_name | Database name to create in RDS for Apiary | string | `apiary` | no |
+| apiary_domain_name | Apiary domain name for route 53 | string | `` | no |
+| apiary_log_bucket | bucket for apiary logs | string | - | yes |
```

Comment:
You start the other "Description"s with an uppercase letter so you should do the same here, similar for many below so the table is consistent.